test/client/index_test.rb

```diff
@@ -270,8 +270,12 @@
       assert_equal 0, response["_shards"]["failed"]
     end
 
-    it "optimizes" do
-      response = @index.optimize
+    it "force merges" do
+      if es_version_5_x?
```

Comment:
Since ES 2 has forcemerge and Elastomer is aliasing it to `optimize` as a courtesy for previous users of this library, I don't think this test needs to be conditionalized -- it should work on either ES version with either method call.

You can test both versions if you like.

Reply:
That makes sense!